lib/elix/responders/lists.ex

```diff
@@ -11,7 +11,7 @@ defmodule Elix.Responders.Lists do
   show lists - Displays all lists
   """
   respond ~r/show lists\Z/i, msg do
-    reply(msg, render_items(List.all_names))
+    send(msg, render_items(List.all_names))
```

Comment:
Not really related, but I should show some kind of "There are no lists yet" message when `List.all_names` is empty. Currently, the robot just doesn't reply.